### ISSUE TITLE
Scrollbar is hidden if previewer theme adds padding to the wrapper

### ISSUE THREAD

**Issue by OscarGodson** [OscarGodson/EpicEditor#13]
It has to do with `_getStyle()` returning `0px` even tho it's _not_ `0px`. If i set the padding with JS like `this.previewer.style.padding = '10px'` then `_getStyle()` magically works, but if it's set with CSS it doesn't. It's also strange because i can get the padding of any other element just fine.

**Comment by OscarGodson** [comment]
I have a shitty partial fix where I have added:

``` javascript
//FIXME Should be in theme!
this.previewer.style['padding'] = '10px'; //HERE
this.previewer.style.width  = this.element.offsetWidth - _outerWidth(this.previewer) - widthDiff +'px';
this.previewer.style.height = this.element.offsetHeight - _outerHeight(this.previewer) - heightDiff +'px';
```

But it should use `_getStyle()` and `padding-left`, `padding-right`, `padding-top`, and `padding-bottom`, but for some reason the computed style is saying 0px.

**Comment by OscarGodson** [comment]
Moving this to public beta since this only affects theming

**Comment by OscarGodson** [comment]
This was finally fixed thanks, again, to #59
